### ISSUE TITLE
LUIS AzureAccount APIs all return 400 because the SDK does not send Authorization parameter

### ISSUE THREAD

**Issue by hansmbakker** [Azure/azure-sdk-for-net#8023]
**Describe the bug**
The [`azureaccounts` endpoints for the LUIS authoring API](https://westeurope.dev.cognitive.microsoft.com/docs/services/5890b47c39e2bb17b84a55ff/operations/5be313cec181ae720aa2b26c/console) all require an `Authorization` header containing `Bearer <ARM token>` but the NuGet package does not supply this.

***Exception or Stack Trace***
`Microsoft.Rest.TransientFaultHandling.HttpRequestWithStatusException: 'Response status code indicates server error: 400 (BadRequest).'`

**To Reproduce**

- Use the [`Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring`](https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring/) nuget package (v3.1.0)
- Create a `LUISAuthoringClient` with the subscriptionKey of your LUIS Authoring Azure resource in the `ApiKeyServiceClientCredentials`:
```csharp
            var credentials = new ApiKeyServiceClientCredentials(authoringKey);
            _luisClient = new LUISAuthoringClient(credentials, new DelegatingHandler[] { });
```
- call `await _luisClient.AzureAccounts.ListUserLUISAccountsAsync()` or any other api making use of the `azureaccounts` endpoint

***Code Snippet***
```csharp
            var credentials = new ApiKeyServiceClientCredentials(authoringKey);
            _luisClient = new LUISAuthoringClient(credentials, new DelegatingHandler[] { });
            await _luisClient.AzureAccounts.ListUserLUISAccountsAsync();
```

**Expected behavior**
The provided functions in the NuGet package work foolproof

**Screenshots**

**Setup (please complete the following information):**
 - OS: Windows
 - IDE : Visual Studio 16.4 Preview 1
 - Version of the Library used: 3.1.0

**Additional context**
Workaround is to manually add the `Authorization` header using the `customHeaders` parameter of `ListUserLUISAccountsWithHttpMessagesAsync` (which results in ugly code and defeats the purpose of ListUserLUISAccountsAsync)

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by kayousef** [comment]
Thanks @hansmbakker for reporting this. I have investigated the issue and found that the header is indeed missing from the function signature. 
Please keep unblocking yourself using the custom header approach. Meanwhile, a fix will be included in the next release which should be coming soon

**Comment by enihcam** [comment]
I don't know whether what i'm having here is the same issue.

I'm using _KeyVaultClient_ plus a custom logging handler, and _Authorization_ header is dropped.

Without a delegate handler, everything works fine.

My workaround is to re-acquire the token in the logging handler, which looks ugly.

**Comment by SatishBoddu-MSFT** [comment]
Hello @hansmbakker Apologies for the delayed response. I see that this issue is opened a long time ago and no further activity had taken place. So wanted to check if you are still looking for assistance on this query? Please let us know.

**Comment by hansmbakker** [comment]
Thank you for getting back to me - I'm not using the package anymore personally - cannot say whether the issue still exists for other people though.

**Comment by JimSuplizio** [comment]
Hi @hansmbakker, we deeply appreciate your input into this project. Regrettably, this issue has remained inactive for over 2 years, leading us to the decision to close it. We've implemented this policy to maintain the relevance of our issue queue and facilitate easier navigation for new contributors. If you still believe this topic requires attention, please feel free to create a new issue, referencing this one. Thank you for your understanding and ongoing support.
